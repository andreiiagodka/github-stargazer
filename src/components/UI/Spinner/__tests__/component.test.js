import { shallow } from 'enzyme'

import SpinnerComponent from '..'

describe('<SpinnerComponent />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SpinnerComponent />)

    expect(wrapper).toMatchSnapshot()
  })
})
