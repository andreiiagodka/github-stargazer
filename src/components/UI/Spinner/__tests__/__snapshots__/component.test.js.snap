// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SpinnerComponent /> renders correctly 1`] = `
<div
  className="Loader"
>
  Loading ...
</div>
`;
